detekt-generator/src/main/kotlin/dev/detekt/generator/collection/RuleVisitor.kt

```diff
@@ -28,8 +27,7 @@ internal class RuleVisitor(textReplacements: Map<String, String>) : DetektVisito
     private val configurationCollector = ConfigurationCollector()
     private val classesMap = mutableMapOf<String, Boolean>()
     private var deprecationMessage: String? = null
-    private val fullAnalysisInterfaces =
-        setOf(RequiresFullAnalysis::class.simpleName, RequiresAnalysisApi::class.simpleName)
+    private val fullAnalysisInterfaces = setOf(RequiresAnalysisApi::class.simpleName)
```

Comment:
The tiniest of nits...

```suggestion
    private val fullAnalysisInterface = setOf(RequiresAnalysisApi::class.simpleName)
```

Reply:
Fixed